scoring_engine/web/views/api/overview.py

```diff
@@ -42,31 +37,28 @@ def overview_get_round_data():
 
 
 @mod.route('/api/overview/data')
-@cache.memoize()
 def overview_data():
     team_data = OrderedDict()
-    teams = session.query(Team).filter(Team.color == 'Blue').order_by(Team.id).all()
-    random.shuffle(teams)
-    for team in teams:
+    blue_teams = Team.get_all_blue_teams_cached()
```

Comment:
I created a new function specifically for the cache, so that we don't muck with other areas that use "Team.get_all_blue_teams"